src/iocore/net/QUICNetProcessor.cc

```diff
@@ -50,7 +50,7 @@ DbgCtl dbg_ctl_iocore_net_processor{"iocore_net_processor"};
 } // end anonymous namespace
 
 static void
-debug_log(const char *line, void * /* argp ATS_UNUSED */)
+debug_log(const char *line)
```

Comment:
I didn't read this carefully - this is actually used as a callback so it needs the parameter.

Reply:
I'm sorry.

Reply:
I just saw the same after I pushed the changes.
Shouldn't have hurried this.
I'm about to revert it back.